### ISSUE TITLE
reference data does not dereference

### ISSUE THREAD

**Issue by cristianvasquez** [epimorphics/IntervalServer#1]
Hello, 

I'm trying to have a look into:

http://reference.data.gov.uk/id/quarter/2016-Q1

But does not dereference

**Comment by der** [comment]
Yes, the UK government ceased to support and fund that initiative and shut down that service some time ago.

**Comment by ajtucker** [comment]
As far as I'm aware, it dropped offline around September last year.

It was being run I think on Heroku by UK gov's GDS, but as it hasn't been supported or updated in a number of years, the plug was finally pulled.

CC: @rossbowen, @BillSwirrl, @RicSwirrl
